.github/workflows/action-build.yml

```diff
@@ -29,44 +39,63 @@ jobs:
         run: yarn
 
       - name: Build application
-        run: yarn build
+        run: APP_ENV=development yarn build
```

Comment:
What if you're building for mainnet? This has to be production

Reply:
moved to deploy repo